StardewArchipelago/Locations/CodeInjections/Vanilla/ShippingInjections.cs

```diff
@@ -111,6 +111,10 @@ private static string GetShippedItemName(Item shippedItem)
             if (name.Contains("moonslime.excavation."))
             {
                 name = shippedItem.DisplayName; //Temporary fix; will break for chinese speaking players only atm
+                if (name.Contains("Woooden"))
+                {
+                    name = name.Replace("Woooden", "Wooden");

```

Comment:
You don't need an if. Running string.Replace(str1, str2) will simply do nothing if str1 is not present